splunk_add_on_ucc_framework/ucc_ui_lib/src/main/webapp/components/table/InputTable.jsx

```diff
@@ -0,0 +1,221 @@
+import React, { useState, useEffect } from 'react';
+import DL from '@splunk/react-ui/DefinitionList';
+import Table from '@splunk/react-ui/Table';
+import Switch from '@splunk/react-ui/Switch';
+import ButtonGroup from '@splunk/react-ui/ButtonGroup';
+import Pencil from '@splunk/react-icons/Pencil';
+import Clone from '@splunk/react-icons/Clone';
+import Trash from '@splunk/react-icons/Trash';
+import Tooltip from '@splunk/react-ui/Tooltip';
+import { _ } from '@splunk/ui-utils/i18n';
+import { ButtonComponent, TitleComponent, SubTitleComponent, TableCaptionComponent } from './InputTableStyle';
+
+function getExpansionRow(colSpan, row) {
+    return (
+        <Table.Row key={`${row.email}-expansion`}>
+            <Table.Cell style={{ borderTop: 'none' }} colSpan={colSpan}>
+                <DL>
+                    <DL.Term>Name</DL.Term>
+                    <DL.Description>{row.name}</DL.Description>
+                    <DL.Term>Interval</DL.Term>
+                    <DL.Description>{row.interval}</DL.Description>
+                    <DL.Term>Index</DL.Term>
+                    <DL.Description>{row.index}</DL.Description>
+                    <DL.Term>Status</DL.Term>
+                    <DL.Description>{String(row.status) ? "Enabled": "Disabled"}</DL.Description>
+                    <DL.Term>Object</DL.Term>
+                    <DL.Description>{row.object}</DL.Description>
+                    <DL.Term>Object Fields</DL.Term>
+                    <DL.Description>{row.object_fields}</DL.Description>
+                    <DL.Term>Order By</DL.Term>
+                    <DL.Description>{row.order_by}</DL.Description>
+                    <DL.Term>Limit</DL.Term>
+                    <DL.Description>{row.limit}</DL.Description>
+                </DL>
```

Comment:
Need to dynamically generate this based on JSON. The current behavior of globaConfig is, if moreInfo is present then add expansion row.

Reply:
Also I think this will require localization. Can we wrap titles in `_()`?

Reply:
Done